spec/controllers/pageflow/editor/file_import_controller_spec.rb

```diff
@@ -13,13 +13,36 @@ module Pageflow
         expect(response.status).to eq(302)
       end
 
-      it 'makes the call to the specified file importer' do
+      it 'does not invoke the disabled importer' do
         user = create(:user)
+        file_importer = FileImporterHelper.new
         entry = create(:entry, with_editor: user)
         sign_in(user, scope: :user)
-        file_importer = FileImporterDummy.new
         pageflow_configure do |config|
-          config.file_importers.register(file_importer.name, file_importer)
+          config.features.register(file_importer.feature_name) do |feature_config|
+            feature_config.file_importers.register(file_importer)
+          end
+        end
+
+        expect(file_importer).not_to receive(:search)
+        expect {
+          get(:search,
+              params: {query: 'test', file_import_name: file_importer.name, entry_id: entry})
+        }.to raise_error
```

Comment:
Always expect an explicit error when `raise_error`. This could be a syntax error that makes the spec pass...